Sources/Instrumentation/Tracing/Span.swift

```diff
@@ -36,7 +36,7 @@ public protocol Span {
     var endTimestamp: Timestamp? { get }
 
     /// The read-only `BaggageContext` of this `Span`, set when starting this `Span`.
-    var baggage: BaggageContext { get }
+    var context: BaggageContext { get }
```

Comment:
IMHO these should be context because span.context rather than span.baggage.

The baggage word is only to be used to disambiguate in cases like context.context so this isn't one of those cases IMO. (rules we arrived at https://github.com/slashmo/gsoc-swift-baggage-context#existing-context-argument )

Reply:
requires major version bump ;-)

Reply:
Agreed 👍

Reply:
The project was never tagged so we're not even a 0.0.0, a major bump from nothing still is nothing 😉